<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IvancinBox</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>600</width>
      <height>450</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="login_page">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>471</width>
        <height>144</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Espera unos segundos mientras se carga la p치gina</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Debes hacer Log in en DropBox</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Dar permiso a la aplicaci칩n</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Copiar el c칩digo de autorizaci칩n:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Click en &quot;Acceder&quot;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>380</y>
        <width>98</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Acceder</string>
      </property>
     </widget>
     <widget class="QLabel" name="errorLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>410</y>
        <width>321</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="dp_page">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>381</width>
        <height>241</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>IvancinBox</string>
       </property>
      </column>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>160</width>
        <height>241</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="subirButton">
         <property name="text">
          <string>Subir archivo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="descargarButton">
         <property name="text">
          <string>Descargar</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="salirButton">
         <property name="text">
          <string>Salir</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
